gradle/verification-metadata.xml

```diff
@@ -411,11 +411,41 @@
             <sha256 value="0addec670fedcd3f113c5c8091d783280d23f75e3acb841b61a9cdb079376a08" origin="Verified" reason="Artifact is not signed"/>
          </artifact>
       </component>
+      <component group="cglib" name="cglib" version="3.2.6">
+         <artifact name="cglib-3.2.6.jar">
+            <sha256 value="2d1f0f1bcc45738c9741cbbe3ae0de4c7724e51193ba6aa97d582d4e7e842ae8" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="cglib" name="cglib-nodep" version="2.1_3">
          <artifact name="cglib-nodep-2.1_3.jar">
             <sha256 value="e77f0b091a800acd15217bbc3c3629c1af0925b55b271fc5d0586422985639bb" origin="Verified" reason="Artifact is not signed"/>
          </artifact>
       </component>
+      <component group="com.amazonaws" name="aws-java-sdk-core" version="1.12.780">
```

Comment:
this stuff seems to be broken 4 sure.

doing this change is head should not apply any changes, right?

But it does so the process is broken rendering this stuff outdated!!!

or what am i doing wrong? @ov7a